torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -42,14 +42,15 @@ def resize_image_tensor(
     max_size: Optional[int] = None,
     antialias: Optional[bool] = None,
 ) -> torch.Tensor:
+    # TODO: use _compute_output_size to enable max_size option
+    max_size  # ununsed right now
```

Comment:
It's unclear to me why we remove `max_size` from here. Shouldn't the TODO be address in this PR?

Reply:
I'll do that in another PR with appropriate tests. Just splitting into 2 PRs and make CI pass.